src/util/make-knex.js

```diff
@@ -14,6 +14,14 @@ export default function makeKnex(client) {
   // The object we're potentially using to kick off an initial chain.
   function knex(tableName, options) {
     const qb = knex.queryBuilder()
+
+    if (qb.client.config.asyncStackTraces) {
```

Comment:
This unfortunately won't work for `SchemaBuilder` or `Raw` since this function is strictly `QueryBuilder`. 

This should result in that `_asyncStack` does not exist in the `.then` handler change [here](https://github.com/tgriesser/knex/pull/2505/files#diff-ff9eb136561ea616d243650733d24677) for `SchemaBuilder` / `Raw`.

Reply:
right, I'll add the `_asyncStack` generatin for SchemaBuilder and for Raw too. Thanks!